pilot/platform/kube/inject/testdata/auth.non-default-service-account.yaml.injected

```diff
@@ -53,7 +51,7 @@ spec:
         - --proxyAdminPort
         - "15000"
         - --controlPlaneAuthPolicy
-        - NONE
+        - MUTUAL_TLS
```

Comment:
Did the default change for off to switch to on?

Reply:
Good catch. I used the deprecated `MeshConfig.AuthPolicy` instead of `ProxyConfig.ControlPlaneAuthPolicy` in the templates by mistake.